src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -46,10 +54,11 @@
     public static final Integer DEFAULT_RETENTION_TIME_MINUTES = (DEFAULT_LAUNCH_TIMEOUT_SECONDS / 60) + 1;
     public static final String DEFAULT_RUN_AS_USER = "jenkins";
     public static final String ERROR_NO_SUBNETS = "No subnetworks exist in the given network and region.";
-    public static final String METADATA_STARTUP_SCRIPT_KEY = "startup-script";
+    public static final String METADATA_LINUX_STARTUP_SCRIPT_KEY = "startup-script";
+    public static final String METADATA_WINDOWS_STARTUP_SCRIPT_KEY = "windows-startup-script-ps1";
     public static final String NAT_TYPE = "ONE_TO_ONE_NAT";
     public static final String NAT_NAME = "External NAT";
-    public static final List<String> KNOWN_LINUX_IMAGE_PROJECTS = Collections.unmodifiableList(new ArrayList<String>() {{
+    public static final List<String> KNOWN_IMAGE_PROJECTS = Collections.unmodifiableList(new ArrayList<String>() {{
```

Comment:
Suggest using ImmutableList for conciseness, see: https://google.github.io/guava/releases/21.0/api/docs/com/google/common/collect/ImmutableList.Builder.html

Reply:
Using the type ImmutableList also makes it clearer that the list is not meant to be changed.